packages/core/src/button/button.element.spec.ts

```diff
@@ -100,20 +111,42 @@ describe('button element', () => {
       expect(o.f).not.toHaveBeenCalled();
     });
 
-    it('should not interact with form elements if type is button', async () => {
```

Comment:
This test was duplicate of the one before it. That's why it looks overwritten here.